### DISCUSSION TITLE
Ordering Enum entries

### DISCUSSION THREAD

**Discussion by criptomikon** [andyglow/scala-jsonschema#296] [Q&A]
Hi,

I need some help to understand how can I apply the ordering of the Enum to the Json Schema. 

From this object:

`sealed trait EnumToOrder extends EnumEntry with Ordered[EnumToOrder]` where the compare method is implemented, I create the case objects like this:

```
object EnumToOrder extends Enum[EnumToOrder] {
     implicitly[Ordering[EnumToOrder]]
     val values = findValues.sorted

     case object Enum1 extends EnumToOrder {
        ....
     }

    case object Enum2 extends EnumToOrder {
        ....
     }


     ....
```

This is used in another object like this:

```
  final case class AnotherObject(
                              fieldName: String,
                              enumObject: EnumToOrder){
```

For some reason I don't understand, when generating the Json Schema, the order is ignored:

```
     "enumObject" : {
        "type" : "string",
        "enum" : [
          "Enum2",
          ....
       ]
```

Is there a way to enforce the order of the possible enum entries in the generated Json Schema?

**Comment by andyglow** [comment]
hello, @criptomikon 
may I ask, what benefits you expect from having them sorted?
